content/setup/ctd.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Set Up a Compute-to-Data enviroment
+description: Set Up a Compute-to-Data enviroment.
+---
+
+
+## Requirments
+
+At the time of writing, we need the following:
+
```

Comment:
Let's start with the folder structure, otherwise this is a bit confusing:

```suggestion
First, create a folder with the following structure:

```
ocean/
  barge/
  operator-service/
  operator-engine/
```
```

Reply:
Damn nested code blocks, should look rendered like this:

First, create a folder with the following structure:

```
ocean/
  barge/
  operator-service/
  operator-engine/
```